kevm-pyk/src/kevm_pyk/foundry.py

```diff
@@ -684,8 +746,8 @@ def foundry_list(foundry_root: Path) -> list[str]:
 def foundry_remove_node(foundry_root: Path, test: str, node: NodeIdLike) -> None:
     foundry = Foundry(foundry_root)
     apr_proofs_dir = foundry.out / 'apr_proofs'
-    contract_name, test_name = test.split('.')
-    proof_digest = foundry.proof_digest(contract_name, test_name)
+    (contract_name, test_sig) = foundry.unique_sig(test)
+    proof_digest = foundry.proof_digest(contract_name, test_sig)
```

Comment:
This logic seems duplicated across some `foundry_*` commands. Can it be extracted?

Reply:
Thanks! Done